src/entities/Task/model/types/task.ts

```diff
@@ -0,0 +1,26 @@
+export enum StatusEnum {
+    COMPLETED = 'COMPLETED',
+    TO_DO = 'TO_DO',
+    IN_PROGRESS = 'IN_PROGRESS'
+}
+
+export interface SubTask {
+    id: string;
+    taskId: string;
```

Comment:
sub tasks exist inside their parent task, doesn't look like the `taskId` property is required

Reply:
Fixed